app/static/styles/main.css

```diff
@@ -1,103 +1,201 @@
-:root {
-  --primary: #879d95;
-  --secondary: #a2799a;
-  --deep: #1b5862;
-  --light: #f8d3ba;
-  --focus: #f76f6c;
+@media (prefers-color-scheme: light) {
+  :root {
+    --primary: #c68bc7;
+    --secondary: #bf936b;
+    --navbar: #203857;
+    --warning: #ed524e;
+    --focus: #74b38b;
+    --text: black;
+    --bg: white;
+  }
```

Comment:
Probably worth looking at further down the track, but I think a second text colour would be good for buttons - buttons might be "dark" themselves so i think white text might look better even in light mode.

And I'm not a huge fan of the colour scheme (especially dark mode), but that can be easily altered later

Reply:
~~i love the dark mode colour scheme and dislike the light mode one currently, subjectivity is problematic~~
I see your point, but i think colour alterations should defs happen last. An alternate button colour only is necessary on light mode also but yeah. Colours were the last thing I changed and are very easy to change now (hopefully)